### DISCUSSION TITLE
Fixed wing - Automated landing at the last waypoint.

### DISCUSSION THREAD

**Discussion by DehnDK** [iNavFlight/inav#7373] [General]
Hello
Running Inav 3.01 on a Matek F405-Wing (New edition).

What I am trying to do is to create a mission, with automated landing at the last waypoint.
I can get it to fly the mission, but it will just loiter at the last wayppoint.

Am I doing something wrong or is it impossible to get the plane to land at the end of the mission?

**Comment by breadoven** [comment]
Should just be a case of setting the last WP as a Land WP:

https://github.com/iNavFlight/inav/wiki/MSP-Navigation-Messages

**Reply by DehnDK** [reply]
I know that would sound like the thing to do, and that is what I did, but it just kept loitering at the last WP.

**Comment by breadoven** [comment]
Can you post the .mission file or the WP output when you type **WP** into the CLI ?

**Reply by DehnDK** [reply]
> Can you post the .mission file or the WP output when you type **WP** into the CLI ?

Sure.
[Airfield.zip](https://github.com/iNavFlight/inav/files/7044130/Airfield.zip)

**Comment by stronnag** [comment]
Works as expected last time I tried it. Plane attempted to land at requested WP, pilot switched to manual to avoid actually 'landing'.

**Comment by DehnDK** [comment]
Could you post your mission file please😊

tir. 24. aug. 2021 kl. 18.13 skrev Jonathan Hudson ***@***.***
>:

> Works as expected last time I tried it. Plane attempted to land at
> requested WP, pilot switched to manual to avoid actually 'landing'.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/iNavFlight/inav/discussions/7373#discussioncomment-1228346>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOCLRY3QVDCNO4INYWLTKVTT6PAJRANCNFSM5CVHMUXA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
-- 
Venlig hilsen / Best regards
Torben Dehn


Banggood.com, stedet hvor alt kan købes.
<https://www.banggood.com/index.php?zf=3683739>

Hobbyking - If you need RC stuff in a hurry
<https://hobbyking.com/en_us/customer/account/create/referral/WNZM4MTGWMCAXNT/>

PayPal.Me <https://www.paypal.me/TorbenDehn>

**Reply by stronnag** [reply]
Sure, for what it's worth. Slightly relocated to protect the guilty. But trivially obvious, set the WP to LAND and the vehicle lands ... unconditionally.

[landing-missions.tar.gz](https://github.com/iNavFlight/inav/files/7040844/landing-missions.tar.gz)

**Comment by DehnDK** [comment]
Hmmmm if i set it to land, it just loiter.
Very Strange..

tir. 24. aug. 2021 kl. 19.04 skrev Jonathan Hudson ***@***.***
>:

> Sure, for what it's worth. Slightly relocated to protect the guilty. But
> trivially obvious, set the WP to LAND and the vehicle lands ...
> unconditionally.
>
> landing-missions.tar.gz
> <https://github.com/iNavFlight/inav/files/7040844/landing-missions.tar.gz>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/iNavFlight/inav/discussions/7373#discussioncomment-1228604>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOCLRY42OC4LXE4B74ES4BLT6PGLNANCNFSM5CVHMUXA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
-- 
Venlig hilsen / Best regards
Torben Dehn


Banggood.com, stedet hvor alt kan købes.
<https://www.banggood.com/index.php?zf=3683739>

Hobbyking - If you need RC stuff in a hurry
<https://hobbyking.com/en_us/customer/account/create/referral/WNZM4MTGWMCAXNT/>

PayPal.Me <https://www.paypal.me/TorbenDehn>

**Comment by breadoven** [comment]
The reason it won't Land is because you have the final WP set to RTH with NO landing option set ... so it just loiters.

To set a landing either use RTH with Parameter 1 = 1 OR set the final WP Action as "LAND" ... as was mentioned in the previous posts.

**Reply by DehnDK** [reply]
> The reason it won't Land is because you have the final WP set to RTH with NO landing option set ... so it just loiters.
> 
> To set a landing either use RTH with Parameter 1 = 1 OR set the final WP Action as "LAND" ... as was mentioned in the previous posts.

Ahhh damn old version, the latest one has LAND as action, but it still Loiter.
